backend/src/controllers/memberController.ts

```diff
@@ -14,21 +14,27 @@ const addMembers = async (req: Request, res: Response) => {
 
   try {
     const membersToCreate = membersData.map(
-      (member: { name: string; walletAddress: string }) => ({
-        name: member.name,
-        walletAddress: member.walletAddress,
-        teamId: Number(id), // team ID
-      }),
+      (member: { name: string; walletAddress: string }) => {
+        if (!isWalletAddressValid(member.walletAddress)) {
+          throw new Error(`Invalid wallet address for member: ${member.name}`);
+        }
+
+        return {
+          name: member.name,
+          walletAddress: member.walletAddress,
+          teamId: Number(id), // team ID
+        };
+      }
     );
 
     const createdMembers = await prisma.member.createMany({
       data: membersToCreate,
     });
 
     res.status(201).json(createdMembers);
-  } catch (error) {
+  } catch (error: any) {
     console.error("Error adding members:", error);
-    res.status(500).json({ error: "Internal server error" });
+    res.status(500).json({ error: `${error.message}` });
```

Comment:
`errorResponse` util takes care of this for you including conditionally printing in dev mode. We standardised to avoid leaking sensitive information about the back-end architecture to the front-end